src/java/src/main/java/com/dnanexus/DXSearch.java

```diff
@@ -989,14 +1013,111 @@ private T getDataObjectInstanceFromResult(FindDataObjectsResponse.Entry e) {
             return (T) dataObject;
         }
 
-
         @Override
         public Iterator<T> iterator() {
             return new ResultIterator();
         }
     }
 
     /**
+     * The page subset of data objects that matched a {@code findDataObjects} query.
+     *
+     * @param <T> data object class to be returned
+     */
+    @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
This no longer needs to be Jackson-serializable, I think, so this annotation could be removed.